### PR TITLE
IBX-156 Submodel preview and other changes

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | IBX-156 |
| Versions      | master |

Describe in detail how users can preview scenario results, incl. ones that have submodels defined.
Also, add a teaser on result integration and update the landing page.

### CODE REVIEW

**Review by dkorostynski — APPROVED** [review]

File: docs/personalization/integrating_results.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Integrating scenario results
+
+When the personalization service is [enabled](enabling_personalization.md) and properly 
+[configured](perso_configuration.md), as an editor, you can embed the recommendations 
+that come from the service, to show them to the end users.
+You can, for example, modify the landing page to include a block that shows what
```
Comment (DominikaK) [inline docs/personalization/integrating_results.md:6]:
```suggestion
You can, for example, modify a Page to include a block that shows what 
```

File: docs/personalization/integrating_results.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Integrating scenario results
+
+When the personalization service is [enabled](enabling_personalization.md) and properly 
+[configured](perso_configuration.md), as an editor, you can embed the recommendations 
+that come from the service, to show them to the end users.
+You can, for example, modify the landing page to include a block that shows what 
+content items/products are recommended to end users when specific [scenarios](scenarios.md) are triggered.
```
Comment (DominikaK) [inline docs/personalization/integrating_results.md:7]:
```suggestion
Content items/products are recommended to end users when specific [scenarios](scenarios.md) are triggered. 
```

File: docs/personalization/integrating_results.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+You can, for example, modify the landing page to include a block that shows what 
+content items/products are recommended to end users when specific [scenarios](scenarios.md) are triggered. 
+Depending on the scenario type, you may need to provide additional information 
+to see the recommendation results.
+
+The number and selection of available scenarios depends on the arrangements that 
+your organization makes with Ibexa when defining the initial configuration.
+
+1. Navigate to the **Site** > **List** tab, and then click the **Preview** 
+icon next to a site, in which you want to place a personalization block.
```
Comment (DominikaK) [inline docs/personalization/integrating_results.md:15]:
```suggestion
icon next to the site in which you want to place a personalization block.
```

File: docs/index.md [hunk 4]
```diff
@@ -59,10 +59,10 @@
                         </h5>
                     <div class="tile-body">
                         <ul>
-                            <li><a href="personalization/personalization/">Personalization</a></li>
-                            <li><a href="personalization/dashboard/">Dashboard</a></li>
-                            <li><a href="personalization/scenarios/">Scenarios</a></li>
-                            <li><a href="personalization/recommendation_models/">Models</a></li>
+                            <li><a href="personalization/personalization/">Introduction</a></li>
+                            <li><a href="personalization/use_cases/">Use cases</a></li>
+                            <li><a href="personalization/perso_configuration/">Configuring</a></li>
```
Comment (DominikaK) [inline docs/index.md:64]:
```suggestion
                            <li><a href="personalization/perso_configuration/">Configuration</a></li>
```

File: docs/personalization/integrating_results.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+1. On the Page Builder screen, click the **Preview/Edit** selector to switch to editing mode.
+
+1. From the **Elements** menu, drag and drop the **Personalized** block to a location on the page layout.
```
Comment (DominikaK) [inline docs/personalization/integrating_results.md:19]: Side note: the block should also be listed in https://doc.ibexa.co/projects/userguide/en/master/site_organization/working_with_page/#block-settings

File: docs/personalization/integrating_results.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+ 
+1. Click the **Block settings** icon to modify the **Personalized** block:
+ 
+    1. On the **Basic** tab, set values in the following fields:
+        -	**Block name** – Optionally, enter a name for the page block, for example, “Bestsellers”.
```
Comment (DominikaK) [inline docs/personalization/integrating_results.md:24]:
```suggestion
        -	**Block name** – Optionally, enter a name for the page block, for example, "Bestsellers".
```

File: docs/personalization/integrating_results.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+        -	**Select a scenario** – Select "Landing page" or "Top clicked".
+        -	**Select a Content Type...** – Select "Product".
+        -	**Display limit** – Set the number of products to be displayed, for example, 4.
+        
+    1. On the **Design** tab, in the **View** field, change the layout to "Products" and submit your changes.
+    
+      The preview of your landing page changes to display a list of products recommended by the personalization service.
```
Comment (DominikaK) [inline docs/personalization/integrating_results.md:31]:
```suggestion
      The preview of the Page changes to display a list of products recommended by the personalization service.
```

File: docs/personalization/integrating_results.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+ 
+1. Save your changes to the draft or publish the landing page.
```
Comment (DominikaK) [inline docs/personalization/integrating_results.md:33]:
```suggestion
1. Save your changes to the draft or publish the Page.
```

File: docs/personalization/previewing_scenario.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
+# Previewing scenario results
+
+If your [user Role](../site_organization/organizing_the_site.md#permissions) includes 
+the `Personalization/View` Policy, you can see what content items/products are recommended
```
Comment (DominikaK) [inline docs/personalization/previewing_scenario.md:4]:
```suggestion
the `Personalization/View` Policy, you can see what Content items/products are recommended 
```

File: docs/personalization/previewing_scenario.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+1. Navigate to the **Personalization** > **Scenarios** tab, and then click the **Preview** 
+icon next to a scenario that you want to preview.
+
+1. If your scenario is based on models of [popularity type](recommendation_models.md#popularity-models), such as, for example, 
+**Landing page** or **Top clicked**, skip to the last step. 
+
+    No further configuration is required.
+
+1. If your scenario is based on models of [collaborative type](recommendation_models.md#collaborative-models), such as, for example, 
+**Also clicked**, in the **Context items** area, click **Add items**, and select a content item/product from the content browser:
```
Comment (DominikaK) [inline docs/personalization/previewing_scenario.md:21]:
```suggestion
**Also clicked**, in the **Context items** area, click **Add items**, and select a Content item/product from the content browser:
```

File: docs/personalization/previewing_scenario.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+
+    1. Navigate to the content item/product to be used as a context item for your recommendation results, for example, **Sites** > **My site** > **Products** > **Category 1** > **Product 1**.
```
Comment (DominikaK) [inline docs/personalization/previewing_scenario.md:23]:
```suggestion
    1. Navigate to the Content item/product to be used as a context item for your recommendation results, for example, **Sites** > **My site** > **Products** > **Category 1** > **Product 1**.
```

File: docs/personalization/previewing_scenario.md [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+    1. Click the **Add** icon to select the content item/product, and then click the **Confirm selection** icon.
```
Comment (DominikaK) [inline docs/personalization/previewing_scenario.md:24]:
```suggestion
    1. Click the **Add** icon to select the Content item/product, and then click the **Confirm selection** icon.
```

File: docs/personalization/previewing_scenario.md [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+area, click **Select path**, and select a category to filter the results:
+
+    1. Navigate to the category to be used as a filter, and then click the **Confirm selection** icon.
+
+1. If your collaborative scenario uses the end user’s history as context, like, for example, 
+**Also clicked - user**, enter an end user identifier in the **User id** field, for example, 500.
+
+1. If your scenario has the use of [submodels](recommendation_models.md#submodels) enabled, 
+in the **Custom parameters** field, enter the phrase that defines a set of items 
+based on a specific attribute, for example "material=wood”, and then click the **Add** button.
```
Comment (DominikaK) [inline docs/personalization/previewing_scenario.md:37]:
```suggestion
based on a specific attribute, for example "material=wood", and then click the **Add** button.
```

File: docs/personalization/previewing_scenario.md [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+
+1. Click **Send request** to display the results.
+
+!!! note "Code display"
```
Comment (DominikaK) [inline docs/personalization/previewing_scenario.md:41]:
```suggestion
!!! note "Display response"
```

File: docs/personalization/recommendation_models.md [hunk 15]
```diff
@@ -138,16 +138,16 @@ The item catalog must be filled separately as a result of [data import](conten
 
 Statistics-based recommendations often have the disadvantage of providing recommendations limited 
 to the most popular, most suitable to the user, or most similar products. 
-You might need to extend the set of available recommendations by defining a subset of 
+You might wantd to extend the set of available recommendations by defining a subset of
```
Comment (DominikaK) [inline docs/personalization/recommendation_models.md:141]:
```suggestion
You might want to extend the set of available recommendations by defining a subset of 
```

File: mkdocs.yml [hunk 16]
```diff
@@ -39,6 +39,8 @@ nav:
         - 'Use cases': 'personalization/use_cases.md'
         - 'Enabling personalization': 'personalization/enabling_personalization.md'
         - 'Configuring personalization': 'personalization/perso_configuration.md'
+        - 'Previewing scenario results': 'personalization/previewing_scenario.md'
+        - 'Integrating scenario results': 'personalization/integrating_results.md'
         - 'Importing data': 'personalization/content_import.md'
         - 'Reviewing performance': 'personalization/dashboard.md'
 #        - 'Embedding personalization results':
```
Comment (DominikaK) [inline mkdocs.yml:46]:
```suggestion
```
Covered in "Previewing"?

**Review by DominikaK — APPROVED** [review]

File: mkdocs.yml (hunk 16 above)
Comment (dabrt) [reply mkdocs.yml:46]: Covered indeed, but in "Integrating scenario results"

**Pull request merged**